.github/workflows/first-time-setup.yml

```diff
@@ -0,0 +1,66 @@
+name: first-time-setup
+run-name: Setup
+
+on:
+  # Run when repo is generated from template
+  create:
+
+# Only keep latest run of this workflow
+concurrency:
+  group: first-time-setup
+  cancel-in-progress: true
+
+permissions: 
+  actions: write
+  checks: write
+  contents: write
+
+
+jobs:
+  first-time-setup:
+    # Ensure run only once, when repo generated
+    if: github.run_number == 1
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Remove first time setup workflow
+        run: rm -f .github/workflows/first-time-setup.yml
+
+      - name: Remove declarations
+        run: rm -f declarations/*
+
+      - name: Update dependabot
+        run: |
+          sed -i "s|"OpenTermsArchive/demo"|"${{ github.repository_owner }}/${{ github.event.repository.name }}"|g" .github/dependabot.yml
+
+      - name: Update config
+        run: |
+          sed -i "s|\"name\": \"demo\"|\"name\": \"${{ github.event.repository.name }}\"|g" config/production.json &&
+          sed -i "s|demo-declarations|${{ github.event.repository.name }}|g" config/production.json &&
+          sed -i "s|demo-snapshots|${{ github.event.repository.name }}-snapshots|g" config/production.json &&
+          sed -i "s|demo-versions|${{ github.event.repository.name }}-versions|g" config/production.json &&
+          sed -i "s|OpenTermsArchive|${{ github.repository_owner }}|g" config/production.json
+
+      - name: Update README
+        run: |
+          sed -i "s|# Open Terms Archive - demo|# Open Terms Archive - ${{ github.event.repository.name }}|g" README.md &&
+          sed -i "s|https://github.com/OpenTermsArchive/demo-versions|https://github.com/${{ github.repository_owner }}/${{ github.event.repository.name }}-versions|g" README.md &&
+          sed -i "s|\`OpenTermsArchive/demo-versions\`|\`${{ github.repository_owner }}/${{ github.event.repository.name }}\`|g" README.md &&
+          sed -i "s|<!-- customize this description to your context -->.*<!-- until here -->|_ _ _ _ _ _ _ _ _ _ _ ✍️|g" README.md &&
```

Comment:
This does not scale. I don't think we need to explicitly define what is being customized and we can use generic comment tags. If you believe the value added by defining in the comment tag what is being replaced, let's at least use a broader regexp that enables genericity, such as:
```suggestion
          sed -i "s|<!-- customize [^-] -->.*<!-- until here -->|_ _ _ _ _ _ _ _ _ _ _ ✍️|g" README.md &&
```

The goal with this system of HTML comments was precisely to avoid having to update the action each time we change some metadata, and this is getting dangerously close to it again 😉